src/Setup/DevDivInsertionFiles/BuildDevDivInsertionFiles.vb

```diff
@@ -50,6 +50,14 @@ Public Module BuildDevDivInsertionFiles
             "codeAnalysisService.servicehub.service.json",
             "remoteHostService.servicehub.service.json",
             "serviceHubSnapshotService.servicehub.service.json",
+            "Microsoft.Build.Conversion.Core.dll",
+            "Microsoft.Build.dll",
+            "Microsoft.Build.Engine.dll",
+            "Microsoft.Build.Framework.dll",
+            "Microsoft.Build.Tasks.Core.dll",
+            "Microsoft.Build.Utilities.Core.dll",
+            "Microsoft.VisualStudio.Threading.dll",
+            "Microsoft.VisualStudio.Validation.dll",
```

Comment:
this basically means, we don't put these in our wxs setup authoring file. our service hub authoring (service hub json file) uses vsix mechanism and only that mechanism. which means OOP installed through wxs won't work anyway since servicehub won't discover us.

since we use vsix mechanism, it doesn't matter how it got installed. as long as vsix is installed, OOP will work.

in preview 4, since willow uses vsix as a way to install our component, it will work just fine.

except, there was some change which cause us to include all our dependency explicitly which we would have gotten implicitly in preview 3.